src/index.tsx

```diff
@@ -89,4 +90,9 @@ const App: React.FC = () => {
   )
 }
 
-ReactDOM.render(<App />, document.getElementById('root'))
+ReactDOM.render(
+  <SystemPrefsProvider>
```

Comment:
Hmm, I think it should be in the App

Reply:
It needs to be outside it, since "App" calls "usePrefs", which it can only do within the Provider.

(I did try a few variations of this, and this structure was the best I could come up with -- open to suggestions though. But the main requirement is that `usePrefs` must be called outside the Language provider, since the default language code needs to be passed to the LanguageProvider)

Ideally it would go:
```
<PrefsProvider>
  <ApolloProvider>
    <ToastProvider>
      <LanguageProvider>
```
But we need to make sure that "Prefs" is loaded before rendering all the rest, which I couldn't see how to do without using the `usePrefs` hook and waiting for `loading` to complete, which really needs to happen in "App" as far as I can tell.

Reply:
Ok, oops should have checked where userPrefs is used. Sounds like a change for later, but yeah if we usePrefs within LanguageProvider, it would work